packages/tooltip/README.md

```diff
@@ -55,20 +55,19 @@ import Tooltip from '@leafygreen-ui/tooltip';
+| `shouldClose`  | `function`                                     | Callback that should return a boolean that determines whether or not the `<Tooltip />` should close when a user tries to close it.                                                                                                                                                                                                                                                                                                                                                                                                             | `() => true`           |
+| `align`        | `'top'` \| `'bottom'` \| `'left'` \| `'right'` | Determines the preferred alignment of the `<Tooltip />` component relative to the element passed to the `trigger` prop. If no `trigger` is passed, the Tooltip will be positioned against its nearest parent element.                                                                                                                                                                                                                                                                                                                          | `'top'`                |
+| `justify`      | `'start'` \| `'middle'` \| `'end'` \| `'fit'`  | Determines the preferred justification of the `<Tooltip />` component (based on the alignment) relative to the element passed to the `trigger` prop. If no `trigger` is passed, the Tooltip will be positioned against its nearest parent element.                                                                                                                                                                                                                                                                                             | `'start'`              |
+| `trigger`      | `React.ReactNode`                              | A `React.ReactNode` against which the `<Tooltip />` will be positioned, and what will be used to trigger the opening and closing of the `Tooltip` component, when the `Tooltip` is uncontrolled. If no `trigger` is passed, the `Tooltip` will be positioned against its nearest parent element. If using a `ReactNode` or inline function, trigger signature is: ({children, ...rest}) => (<button {...rest}>trigger {children}</button>). When using a function, you must pass `children` as an argument in order for the tooltip to render. |                        |
```

Comment:
oh, we're also missing that trigger can be a function

Reply:
👍